test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13832,10 +13832,10 @@ def add_mom_dyn_res(self, custom_resource, script_body=None,
         else:
             if dirname is None:
                 dirname = self.pbs_conf['PBS_HOME']
+            local_host = socket.gethostname()
```

Comment:
do we need to pass localhost in create_temp_file? I think localhost is default right?

Reply:
Removed host in create_temp_file call as we always want to create this file on local host by default and copy to remote host or to the location on localhost where user specifies or to PBS_HOME/